enterprise/metrics/src/main/java/org/neo4j/metrics/MetricsSettings.java

```diff
@@ -153,4 +153,12 @@
 
     @Description( "The reporting interval for Graphite. That is, how often to send updated metrics to Graphite." )
     public static Setting<Duration> graphiteInterval = setting( "metrics.graphite.interval", DURATION, "3s" );
+
+    // Prometheus settings
+    @Description( "Set to `true` to enable the Prometheus endpoint" )
+    public static Setting<Boolean> prometheusEnabled = setting( "metrics.prometheus.enabled", BOOLEAN, FALSE );
+
+    @Description( "The hostname and port to use as Prometheus endpoint" )
+    public static Setting<HostnamePort> prometheusEndpoint =
+            setting( "metrics.prometheus.endpoint", HOSTNAME_PORT, "localhost:8080" );
```

Comment:
wouldn't say that 8080 is a good default port value, why not to use next port after default graphite server (2004)?

Reply:
It's an HTTP server, they typically reside in the 8000 range. 8080 was taken from Prometheus example configurations, 2004 sound equally good.